app-emulation/virtualbox-modules/virtualbox-modules-6.1.34.ebuild

```diff
@@ -50,6 +50,9 @@ src_prepare() {
 	if use pax-kernel && kernel_is -ge 3 0 0 ; then
 		eapply -p0 "${FILESDIR}"/${PN}-5.2.8-pax-const.patch
 	fi
+	if kernel_is -ge 5 18 0 ; then
```

Comment:
It shouldn't be necessary to check the kernel version in the ebuild as the preprocessor includes a conditional check, i.e.,  `#if RTLNX_VER_MIN(5,18,0)`.  You could just change the patch from
```
--- vboxnetflt/linux/VBoxNetFlt-linux.c
+++ vboxnetflt/linux/VBoxNetFlt-linux.c
```
to
```
--- a/vboxnetflt/linux/VBoxNetFlt-linux.c
+++ b/vboxnetflt/linux/VBoxNetFlt-linux.c
```
and add `PATCHES=( "${FILESDIR}"/${PN}-5.2.8-pax-const.patch" )` in global scope instead.

Reply:
Thanks, I added ` PATCHES=( "${FILESDIR}"/${PN}-6.1.34-remove-netif_rx_ni.patch )`, it looks better.